tests/unit/core/i18n/test_utils.js

```diff
@@ -509,4 +510,73 @@ describe(__filename, () => {
       expect(i18n.moment('1988-09-22').format('ll')).toEqual('22 sept. 1988');
     });
   });
+
+  describe('formatFilesize', () => {
+    const _formatFilesize = ({
+      _filesize,
+      _log,
+      i18n = fakeI18n(),
+      size = 123,
+    }) => {
+      return utils.formatFilesize({ _filesize, _log, i18n, size });
+    };
+
+    it('receives an expected string from filesize', () => {
+      // This is a sanity check, verifying that filesize does in fact return
+      // what we expect.
+      // Without it filesize could break and we'd never know.
+      const size = 123;
+      expect(_formatFilesize({ size })).toEqual(`${size} B`);
+    });
+
+    it('formats the number returned by filesize', () => {
+      const size = 1234;
+      const _filesize = sinon.stub().returns(`${size} B`);
+      expect(_formatFilesize({ _filesize, size })).toEqual('1,234 B');
+    });
+
+    it('returns the size for an invalid string from filesize', () => {
+      const size = 987;
+      const _filesize = sinon.stub().returns('123');
+      expect(_formatFilesize({ _filesize, size })).toEqual(size.toString());
+    });
+
+    it('logs an error for an invalid string from filesize', () => {
+      const fakeLog = {
```

Comment:
You can use the shiny new `fakeLogger` now:

https://github.com/mozilla/addons-frontend/blob/8649ddcd51b59aaff94c4a1b5ff53803fae10841/tests/unit/helpers.js#L1120-L1128